tests/playwright-tests/playwright.config.ts

```diff
@@ -1,25 +1,29 @@
 import { defineConfig, devices } from '@playwright/test';
 import environment from '@util/env';
 
-const { PUBLIC_USERNAME, PUBLIC_PASSWORD } = environment;
+const { PUBLIC_USERNAME, PUBLIC_PASSWORD, PUBLIC_URL } = environment;
 
-/**
- * Read environment variables from file.
- * https://github.com/motdotla/dotenv
- */
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-require('dotenv').config();
+function appendBasicAuthCredentialsToPublic(pubUrl: string) {
```

Comment:
`pubUrl` isn't being used for anything anymore I don't think, so I think you can get rid of it now.